plugins/hydra_rq_launcher/news/1586.plugin

```diff
@@ -0,0 +1 @@
+Add support for SSL Redis connections in the RQ launcher plugin.
```

Comment:
nit: could you add a newline after the text?
also remove the trailing period (to be consistent with the rest of the news segment :) )

Reply:
19d65bda4a805fce33381ddc51d3c6d1c565e641 👍🏻